tests/test_actions.py

```diff
@@ -640,3 +654,181 @@ def test_describe(self) -> None:
         assert SeeAnyOf().describe() == "See if no tests pass 🤔."
         assert SeeAnyOf(test).describe() == "See if 1 test passes."
         assert SeeAnyOf(*tests).describe() == f"See if any of {len(tests)} tests pass."
+
+
+class SimpleQuestion(Answerable):
+    @beat("{} examines SimpleQuestion")
+    def answered_by(self, actor: Actor):
+        return True
+
+    def describe(self):
+        return "SimpleQuestion"
+
+
+class TestQuietly:
+    def test_can_be_instantiated(self) -> None:
+        q1 = Quietly(None)
+        q2 = Quietly(1)
+        q3 = Quietly(FakeQuestion())
+        q4 = Quietly(FakeAction())
+        q5 = Quietly(FakeResolution())
+        q6 = QuietlyAnswerable(FakeQuestion())
+        q7 = QuietlyPerformable(FakeAction())
+        q8 = QuietlyResolvable(FakeResolution())
+
+        assert q1 is None
+        assert q2 == 1
+        assert isinstance(q3, QuietlyAnswerable)
+        assert isinstance(q4, QuietlyPerformable)
+        assert isinstance(q5, QuietlyResolvable)
+        assert isinstance(q6, QuietlyAnswerable)
+        assert isinstance(q7, QuietlyPerformable)
+        assert isinstance(q8, QuietlyResolvable)
+
+    def test_implements_protocol(self) -> None:
+        q1 = Quietly(FakeQuestion())
+        q2 = Quietly(FakeAction())
+        q3 = Quietly(FakeResolution())
+        q4 = QuietlyAnswerable(FakeQuestion())
+        q5 = QuietlyPerformable(FakeAction())
+        q6 = QuietlyResolvable(FakeResolution())
+
+        assert isinstance(q1, Answerable)
+        assert isinstance(q2, Performable)
+        assert isinstance(q3, Resolvable)
+        assert isinstance(q4, Answerable)
+        assert isinstance(q5, Performable)
+        assert isinstance(q6, Resolvable)
+
+    def test_passthru_attribute(self):
+        a = FakeAction()
+        a.describe.return_value = "Happy Thoughts"
+        assert Quietly(a).describe() == "Happy Thoughts"
```

Comment:
I know i'm a stickler, but can we separate the "Arrange" and "Act" and "Assert" in this test and the ones that follow?

Reply:
See the latest commit.